src/test/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractFileValidationServiceImplTest.java

```diff
@@ -0,0 +1,113 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import static org.junit.Assert.*;
+
+import java.sql.Date;
+import java.util.Calendar;
+
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import org.kuali.kfs.krad.exception.ValidationException;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+import edu.cornell.kfs.concur.batch.businessobject.ConcurRequestExtractFile;
+import edu.cornell.kfs.concur.batch.businessobject.ConcurRequestExtractRequestDetailFileLine;
+import edu.cornell.kfs.concur.batch.businessobject.ConcurRequestExtractRequestEntryDetailFileLine;
+
+public class ConcurRequestExtractFileValidationServiceImplTest {
+    private ConcurRequestExtractFileValidationServiceImpl concurRequestExtractFileValidationService;
+    
+    @Before
+    public void setUp() throws Exception {
+        Logger.getLogger(ConcurRequestExtractFileValidationServiceImpl.class).setLevel(Level.DEBUG);
+        concurRequestExtractFileValidationService = new ConcurRequestExtractFileValidationServiceImpl();
+    }
+    
+    @After
+    public void tearDown() throws Exception {
+        concurRequestExtractFileValidationService = null;
+    }
+    
+    @Test
+    public void testHeaderAmountMatches() {
+        ConcurRequestExtractFile testFile = buildConcurRequestExtractFile();
```

Comment:
We are saying the same thing.  I will be changing the tests to look like:
    @Test
    public void testHeaderAmountMatches() {
        ConcurRequestExtractFile testFile = buildConcurRequestExtractFile();
        assertTrue("Expected Result: Header amount SHOULD match sum of row amounts from file.", concurRequestExtractFileValidationService.fileApprovedAmountsMatchHeaderApprovedAmount(testFile));
    }
    
    @Test
    public void testHeaderAmountDoesNotMatch() {
        ConcurRequestExtractFile testFile = buildConcurRequestExtractFile();
        testFile.setTotalApprovedAmount(new KualiDecimal(9.87));
        assertFalse("Expected Result: Header amount should NOT match sum of row amounts from file.", concurRequestExtractFileValidationService.fileApprovedAmountsMatchHeaderApprovedAmount(testFile));
    }